IFC4x3/Properties/l/Length_33Y8GlIfD6oOm0fH_CAe__/DocProperty.xml

```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="utf-8"?>
-<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Length" UniqueId="c388842f-4a93-46c9-8c00-a51f8c2a8fbe" PrimaryDataType="IfcPositiveLengthMeasure" />
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="Length_33Y8GlIfD6oOm0fH_CAe__" Name="Length" UniqueId="c388842f-4a93-46c9-8c00-a51f8c2a8fbe" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Reuse?

Reply:
I think, I'm already reusing a property.

Reply:
It is already a reuse yes